docs/configuration.md

```diff
@@ -181,10 +181,16 @@ of the most common options to set are:
   <td><code>spark.driver.memoryOverhead</code></td>
   <td>driverMemory * 0.10, with minimum of 384 </td>
   <td>
-    The amount of off-heap memory to be allocated per driver in cluster mode, in MiB unless
-    otherwise specified. This is memory that accounts for things like VM overheads, interned strings, 
-    other native overheads, etc. This tends to grow with the container size (typically 6-10%). 
-    This option is currently supported on YARN, Mesos and Kubernetes.
+    Amount of non-heap memory to be allocated per driver process in cluster mode 
+    (e.g. YARN, Mesos and Kubernetes.), in MiB unless otherwise specified. This is memory that
```

Comment:
This line changed, to mention YARN et al here. It implies that these resource managers always run in cluster mode. They don't, so I would revert this change and restore the sentence starting with "This option..."

Reply:
OK , I have reverted "This option..." and delete "(e.g. "